### ISSUE TITLE
Password Textbox default font type and size differs from Textbox

### ISSUE THREAD

**Issue by gabryk91** [AppyBuilder/AppyBuilder#56]
Password textbox and textbox with default settings looks different

![image](https://user-images.githubusercontent.com/9896533/37252232-eecbcb2a-251d-11e8-87bc-9367bffdfde3.png)

**Comment by jcoleman2005** [comment]
@gabryk91  Are you referring to hint font?

**Comment by gabryk91** [comment]
yes, the difference is there also on entered text, but more noticeable on the hint
